Pokedex/Models/PokemonResponse.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Pokemon.swift
+//  Pokedex
+//
+//  Created by krikaz on 6/9/20.
+//  Copyright © 2020 thewire. All rights reserved.
+//
+
+import Foundation
+
+private struct RawServerResponse: Decodable {
+    
+    var moves: [SingleMove]
+    var sprites: Sprite
+    var types: [SingleType]
+    var name: String
```

Comment:
For network models, you want them to be immutable hence the use of struct rather than class. So you should also use `let` instead of `var`.